payment/payment.go

```diff
@@ -28,19 +28,19 @@ type Payment struct {
 // pub 036f5646ed688b9279369da0a4ad78953ae7e6d300436ca8a3264360efe38236e3
 
 // FromPublicKey creates a Payment struct from a btcec.publicKey
-func FromPublicKey(pubkey *btcec.PublicKey, network *network.Network) Payment {
+func FromPublicKey(pubkey *btcec.PublicKey, network *network.Network) *Payment {
 	publicKeyBytes := pubkey.SerializeCompressed()
 	hash := hash160(publicKeyBytes)[:ripemd160.Size]
 	script := make([]byte, 0)
 	script = append([]byte{Op0}, hash...)
```

Comment:
Just made some tests to cross-check that the segwit addresses produced with go-elements are the same of those returned by the liquidjs-lib.
Of course, the results do not match 😅.
This is because the witness pubkey script misses 1 byte, that is the length of the hash, so this should be changed as follows:
```suggestion
      script = append([]byte{Op0, byte(len(hash))}, hash...)
```


PS. For the test, the expected address is `XZavBojABpfXhPWkw7y9YYFNAezUHZR47m`

Reply:
@sekulicd, please, fix this, then we can merge this 👍

Reply:
Done.